source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -343,45 +374,15 @@ json TransformerInstance::cluster_metadata_callback(const inja::Arguments &args)
     return stringval;
     break;
   }
+  case ProtobufWkt::Value::kStructValue:
+    ABSL_FALLTHROUGH_INTENDED;
```

Comment:
why use fallthrough here and not separate the list and struct cases?

Reply:
This was copy/pasta'd from the upstream JWT filter, I have no strong feeling